docs/plus/rbac.md

```diff
@@ -0,0 +1,121 @@
+# Role Based Access Control
+
+Role Based Access Control(RBAC) helps you restrict actions that can be performed on specific Kubernetes resources. To make this possible, RBAC lets you create roles with rules which define access permissions for your specified resource.
+
+These roles can then be binded to Kubernetes subjects, which could be User, Group or ServiceAccount. 
+
+**NOTE:** Rules or permissions are purely additive and there are no deny rules.
+
+Now, there are two types of roles available,
+* Role: These set permissions within a particular namespace i.e. is for namespaced resources, like, pods, deployments.
+* ClusterRole: These set permissions for non-namespaced resources, like, nodes, urls.
+
+and, similarly there are two types of binding available,
+* RoleBinding: These grant permissions within a specific namespace.
+* ClusterRoleBinding: These grant cluster wide permissions .
+
+!!! tip ""
+    [Role API Reference](../reference/cdk8s-plus-24/typescript.md#role)
+
+!!! tip ""
+    [RoleBinding API Reference](../reference/cdk8s-plus-24/typescript.md#role-binding)
```

Comment:
Mind if we combine these into a list and add a title?

```suggestion
!!! tip "Learn more"
    * [Role API Reference](../reference/cdk8s-plus-24/typescript.md#role)
    * [RoleBinding API Reference](../reference/cdk8s-plus-24/typescript.md#role-binding)
```

Reply:
Makes sense, will make the update.